migrations/versions/2022_050411_5269e8c2c2c9_.py

```diff
@@ -0,0 +1,86 @@
+
+Revision ID: 5269e8c2c2c9
+Revises: 0aaad1740797
+Create Date: 2022-04-12 11:56:30.184292
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+from app.config import CONNECT_WITH_PROTON
+from app.proton.proton_callback_handler import PROTON_PARTNER_ID, PROTON_PARTNER_NAME
+
+
+# revision identifiers, used by Alembic.
+revision = '5269e8c2c2c9'
+down_revision = '0aaad1740797'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    partner_table = op.create_table(
+        "partner",
+        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column("name", sa.String(128), nullable=False),
+        sa.Column("contact_email", sa.String(128), nullable=False, unique=True),
+        sa.Column('created_at', sqlalchemy_utils.types.arrow.ArrowType(), nullable=False, server_default=sa.func.now()),
+        sa.Column('updated_at', sqlalchemy_utils.types.arrow.ArrowType(), nullable=True),
+
+        sa.PrimaryKeyConstraint("id"),
+    )
+
+    partner_api_token_table = op.create_table(
+        "partner_api_token",
+        sa.Column("id", sa.BigInteger(), autoincrement=True, nullable=False),
+        sa.Column("token", sa.String(32), nullable=False, unique=True),
+        sa.Column("partner_id", sa.BigInteger(), nullable=False),
+        sa.Column("expiration_time", sqlalchemy_utils.types.arrow.ArrowType(), nullable=True),
+        sa.Column('created_at', sqlalchemy_utils.types.arrow.ArrowType(), nullable=False),
+        sa.Column('updated_at', sqlalchemy_utils.types.arrow.ArrowType(), nullable=True),
+
+        sa.Index("partner_api_token_token_idx", 'token'),
+        sa.Index("partner_api_token_partner_id_idx", 'partner_id'),
+        sa.ForeignKeyConstraint(['partner_id'], ['partner.id'], ondelete='cascade'),
+        sa.PrimaryKeyConstraint("id"),
+    )
+
+    partner_user = op.create_table(
+        "partner_user",
+        sa.Column("id", sa.BigInteger(), autoincrement=True, nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("partner_id", sa.BigInteger(), nullable=False),
+        sa.Column("partner_email", sa.String(255), nullable=True),
+        sa.Column('created_at', sqlalchemy_utils.types.arrow.ArrowType(), nullable=False),
+        sa.Column('updated_at', sqlalchemy_utils.types.arrow.ArrowType(), nullable=True),
+
+        sa.Index("partner_user_user_id_idx", 'user_id'),
+        sa.Index("partner_user_partner_id_idx", 'partner_id'),
+
+        sa.ForeignKeyConstraint(['partner_id'], ['partner.id'], ondelete='cascade'),
+        sa.ForeignKeyConstraint(['user_id'], ['users.id'], ondelete='cascade'),
+        sa.PrimaryKeyConstraint("id"),
+    )
+
+    op.add_column('users', sa.Column('partner_id', sa.BigInteger(), nullable=True, server_default=None))
+    op.add_column('users', sa.Column('partner_user_id', sa.String(128), nullable=True, server_default=None))
+
+    if CONNECT_WITH_PROTON:
```

Comment:
I think we can remove this as a migration should only contain with DDL (CREATE, DROP, ALTER) and not DML (SELECT, INSERT, UPDATE, DELETE). We can manually create a partner when we enable login with proton.

Reply:
Okay, I'll then remove the hardcoded Proton partner id and run a query on runtime

Reply:
Removed the hardcoded Proton partner id. We will need to manually create the partner on the database